buildpipeline/DotNet-Trusted-Publish.json

```diff
@@ -80,6 +98,26 @@
       }
     },
     {
+        "enabled": true,
+        "continueOnError": false,
+        "alwaysRun": false,
+        "displayName": "Inject signed symbol catalogs",
+        "timeoutInMinutes": 0,
+        "task": {
+            "id": "e213ff0f-5d5c-4791-802d-52ea3e7be1f1",
+            "versionSpec": "1.*",
+            "definitionType": "task"
+        },
+        "inputs": {
+            "scriptType": "inlineScript",
+            "scriptName": "",
+            "arguments": "-ConfigGroup $(ConfigurationGroup) -SymPkgGlob $(AzureContainerSymbolPackageGlob) -PipelineSrcDir $(Pipeline.SourcesDirectory)",
+            "workingFolder": "$(Pipeline.SourcesDirectory)",
+            "inlineScript": "param($ConfigGroup, $SymPkgGlob, $PipelineSrcDir)\nmsbuild build.proj /t:InjectSignedSymbolCatalogIntoSymbolPackages /p:SymbolPackagesToPublishGlob=$PipelineSrcDir\\packages\\AzureTransfer\\$ConfigGroup\\$env:AzureContainerSymbolPackageGlob /p:SymbolCatalogCertificateId=400",
```

Comment:
Correct, this is something that came up and @dagood and I decided was not an issue - for example, CoreFX and CoreCLR are already publishing indexed packages to MyGet while the packages they upload to Azure are not indexed.  Internally we can sign and index the packages at a later time if we need them to be.